modules/end-user-guide/partials/proc_logging-in-to-che-on-openshift-for-the-first-time-registering-as-a-new-user.adoc

```diff
@@ -7,6 +7,11 @@
 
 This section describes how to log in to {prod-short} on OpenShift for the first time registering as a new user.
 
+[NOTE]
+====
+For {prod-short} installed without OpenShift OAuth support, self-registration of users is disabled by default, therefore the option to register a new user is not available on the login page. See xref:administration-guide:configuring-authorization.adoc#enabling-self-registration_{context}[].
```

Comment:
Could it be a prerequisite instead: 

* Self-registration is enabled. See xref:administration-guide:configuring-authorization.adoc#enabling-self-registration_{context}[].

Reply:
Moved